README.md

```diff
@@ -6,13 +6,34 @@ nim-mode
 An emacs major mode for the Nim programming language.
 
 * [Install Nim](http://nim-lang.org/download.html)
-* Install `nim-mode.el` via melpa.
+* Install `nim-mode.el` via MELPA.
 
+## Nimsuggest
+Some packages are depending on [nimsuggest](https://github.com/nim-lang/nimsuggest) (not nim-suggest.el), so if you want to use more integration in Emacs, please visit the link to install nimsuggest.
```

Comment:
If you're talking packages, people will understand "MELPA packages". I don't think you mean that.